modules/vaos/app/models/vaos/va_appointment_request.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+require 'common/models/form'
+
+module VAOS
+  class VAAppointmentRequest < Common::Form
+    include ActiveModel::Validations
+
+    attribute :email, String
+    attribute :phone_number, String
+    attribute :option_date1, String
```

Comment:
We're constrained by the service so this is rhetorical, but if we weren't, could this be an options array?

Reply:
Yes, i'd love to that when the underlying service can support it as such. This is the craziest form object I've ever worked with and I've seen some really bizarre things in the past. It actually requires a flattened version of much of the same data, but balks if that flattened patient data isn't also provided in an object. My plan is to keep the form validations and handle that kloodgy mess in the service so that the FE doesn't need to be exposed to it.